include/perf_watcher.h

```diff
@@ -47,18 +49,24 @@ typedef struct PerfWatcher {
   X(NOCOUNT, "nocount", nocount, NOCOUNT)                                      \
   X(TRACEPOINT, "tracepoint", events, NOCOUNT)                                 \
   X(CPU_NANOS, "cpu-time", nanoseconds, CPU_SAMPLE)                            \
-  X(CPU_SAMPLE, "cpu-sample", count, NOCOUNT)
+  X(CPU_SAMPLE, "cpu-samples", count, NOCOUNT)                                 \
+  X(ALLOC_SAMPLE, "alloc-samples", count, NOCOUNT)                             \
+  X(ALLOC_SPACE, "alloc-space", bytes, ALLOC_SAMPLE)
 
 #define X_ENUM(a, b, c, d) DDPROF_PWT_##a,
 typedef enum DDPROF_SAMPLE_TYPES {
   PROFILE_TYPE_TABLE(X_ENUM) DDPROF_PWT_LENGTH,
 } DDPROF_SAMPLE_TYPES;
 #undef X_ENUM
 
-enum DDPROF_PERFOPEN_CONFIGS {
-  IS_FREQ = 1 << 0,
-  IS_KERNEL = 1 << 1,
-};
+enum DDProfTypeId { kDDPROF_TYPE_CUSTOM = PERF_TYPE_MAX + 100 };
```

Comment:
It should be easier and easier to add new type of events, nice.
A comment around this definition would be nice, explaining that we are defining our own types different from the perf event API.

Reply:
Done